package.json

```diff
@@ -13,26 +13,26 @@
   },
   "homepage": "https://github.com/Microsoft/app-store-vsts-extension",
   "devDependencies": {
+    "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "^1.0.7",
-    "@types/mocha": "^5.2.7",
     "adm-zip": "^0.4.13",
     "del": "^3.0.0",
-    "gulp": "^4.0.0",
-    "gulp-util": "3.0.4",
+    "gulp": "^4.0.2",
     "minimatch": "3.0.4",
-    "minimist": "^1.1.1",
+    "minimist": "^1.2.5",
     "mocha": "5.2.0",
-    "mocha-junit-reporter": "^1.18.0",
+    "mocha-junit-reporter": "^2.0.2",
     "node-uuid": "1.4.6",
+    "plugin-error": "^1.0.1",
     "q": "1.4.1",
     "semver": "4.3.3",
-    "shelljs": "^0.3.0",
+    "shelljs": "^0.8.5",
     "sync-request": "4.1.0",
-    "tfx-cli": "^0.6.4",
-    "tslint": "^3.15.1",
+    "tfx-cli": "^0.11.0",
+    "tslint": "^5.20.1",
     "typescript": "3.2.2",
-    "validator": "3.33.0",
-    "yargs": "^6.4.0"
+    "validator": "13.7.0",
```

Comment:
This is quite huge leap from 3 to 13 major version - could you please double-check that there won't any changes required for this in tasks and behavior will still be the same?

Reply:
Yeah, double checked it - everything looks good. This package is only used in make-util. The signature of `validator.isLength` function has changed, but it has backwards compatibility fallback, so there is no need to change it:
```js
  if (_typeof(options) === 'object') {
    min = options.min || 0;
    max = options.max;
  } else {
    // backwards compatibility: isLength(str, min [, max])
    min = arguments[1] || 0;
    max = arguments[2];
  }
```